_oasis.in

```diff
@@ -6,8 +6,8 @@ Authors:     Dave Scott
 License:     LGPL-2.1 with OCaml linking exception
 Plugins:     META (0.2)
 OCamlVersion: >= 4.01
-AlphaFeatures: ocamlbuild_more_args
-XOCamlbuildPluginTags: package(cppo_ocamlbuild)
+AlphaFeatures: ocamlbuild_more_args, compiled_setup_ml
+XOCamlbuildPluginTags: package(cppo_ocamlbuild), package(bisect_ppx.ocamlbuild)
```

Comment:
The `_oasis` file instrumentation seems to be a bit different from the script we usually use: https://github.com/simonjbeaumont/ocaml-travis-coveralls/blob/master/travis-coveralls.sh#L6 , but I assume it won't cause any issues that this `package(bisect_ppx.ocamlbuild)` tag is unconditionally there?